indexing-hadoop/src/test/java/org/apache/druid/indexer/DeterminePartitionsJobTest.java

```diff
@@ -177,22 +216,21 @@
                     "2014102100,h.example.com,US,251",
                     "2014102100,i.example.com,US,963",
                     "2014102100,j.example.com,US,333",
-                    "2014102100,k.example.com,US,555",
                     "2014102200,a.example.com,CN,100",
-                    "2014102200,b.exmaple.com,CN,50",
+                    "2014102200,b.example.com,CN,50",
                     "2014102200,c.example.com,CN,200",
                     "2014102200,d.example.com,US,250",
                     "2014102200,e.example.com,US,123",
                     "2014102200,f.example.com,US,567",
                     "2014102200,g.example.com,US,11",
                     "2014102200,h.example.com,US,251",
                     "2014102200,i.example.com,US,963",
-                    "2014102200,j.example.com,US,333",
-                    "2014102200,k.example.com,US,555"
```

Comment:
The behavior of single-dim partitions was changed to size up to the max size instead of up to 1 less.

Before, this test would create partitions of sizes [5, 5, 1], which would be converted to [5, 6], so both would be under the max size of 6.

After the change to size up to the max size, this test creates partitions of sizes [5, 5, 1], which would be converted to [5, 6], and then rejected since the last partition exceeds the max size of 5. Thus the last row was removed from the input data, so that the partition sizes would be [5, 5].